src/formBuilder/CardModal.js

```diff
@@ -74,6 +82,38 @@ export default function CardModal({
             });
           }}
         />
+        <div>
+          <h4>
+            Column Size{' '}
+            <a
+              href='https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout'
```

Comment:
I just tried the ui:column field in the example app of the main branch and it doesn't seem to do anything. Am I missing something?

Reply:
After providing a value in column-size, the UISchema should have something like:

```
newInput1: {
    'ui:column': '5'
  }
```

Reply:
@sifumba  The UISchema is updated as expected, but it doesn't seem to do anything to the way the form is rendered in "Form Preview" -- is there expected to be?

Reply:
It's not expected to be @raymond-lam, the framework will only set the value, then the front-end framework being used will translate the value according to it's configuration.